src/mtx/commands/uplink/getExtendedCurrentValues.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * Uplink command to get extended current values like temperature and frequency.
+ *
+ * @packageDocumentation
+ *
+ * @example create command instance from command body hex dump
+ * ```js
+ * import * as getExtendedCurrentValues from 'jooby-codec/mtx/commands/uplink/getExtendedCurrentValues.js';
+ *
+ * // simple response
+ * const bytes = [0x00, 0x43, 0x00, 0x3c];
+ *
+ * // decoded payload
+ * const parameters = getDeviceType.fromBytes(bytes);
+ *
+ * console.log(parameters);
+ * // output:
+ * {
+ *     temperature: 67,
+ *     frequency: 60
+ * }
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/uplink/GetExtendedCurrentValues.md#response)
+ */
+
+import * as command from '../../utils/command.js';
+import CommandBinaryBuffer, {ICommandBinaryBuffer} from '../../utils/CommandBinaryBuffer.js';
+import * as types from '../../types.js';
+import {READ_ONLY} from '../../constants/accessLevels.js';
+
+
+interface IGetExtendedCurrentValuesResponseParameters {
+    /** device temperature */
+    temperature: types.TInt16,
+    /** The frequency of voltage in the power grid */
```

Comment:
```suggestion

    /** the frequency of voltage in the power grid */
```
or the prev description from upper

Reply:
\+